sonar-project.properties

```diff
@@ -2,5 +2,20 @@ sonar.organization=nhsdigital
 sonar.projectKey=NHSDigital_electronic-prescription-service-api
 sonar.host.url=https://sonarcloud.io
 
-sonar.coverage.exlusions=**/jest.config.ts,**/scripts/*,**/*.spec.ts,**/*.spec.tsx,**/cucumber.js,packages/models/**,**/jest/**,**/*.config.js,**/tests/**,packages/tool/e2e-tests/**,packages/tool/site/server/**,packages/tool/site/client/tests/**
-sonar.javascript.lcov.reportPaths=packages/coordinator/coverage/lcov.info,packages/tool/site/client/coverage/lcov.info,packages/e2e-tests/coverage/lcov.info
+sonar.coverage.exclusions=\
+    **/jest.config.ts, \
+    **/scripts/*, \
+    **/*.spec.ts, \
+    **/*.spec.tsx, \
+    **/cucumber.js, \
+    # packages/models/**, \
```

Comment:
Question - can/should these model definitions be tested? `models` contains a bunch of interfaces.